### PR TITLE
fix: bugs in cart and wishlist docs

### PR DESCRIPTION
**Pull request by AdamPawlinski**
### Related Issues

closes #

### Short Description of the PR

Fixing bugs in cart and wishlist documentation, mostly exmples and few description corrects. 

### Pull Request Checklist

- [] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

File: packages/core/docs/guide/cart-and-wishlist.md [hunk 1]
```diff
@@ -140,50 +148,63 @@ To remove an item from the cart use `removeItem` method, and similarly to update
...
+      };
+    }
+  };
 </script>
 ```
 
 ## Removing all cart items at once
 
-To clear cart items (not delete it) use `clear` method.
+To clear cart items (not delete them) use `clear` method.
```
Comment (filrak) [inline packages/core/docs/guide/cart-and-wishlist.md:188]:
```suggestion
To clear cart items (not delete the cart) use `clear` method.
```

File: packages/core/docs/guide/cart-and-wishlist.md [hunk 2]
```diff
@@ -140,50 +148,63 @@ To remove an item from the cart use `removeItem` method, and similarly to update
...
+    <ul>
+      <li
+         v-for="product in products" :key="product.id"
+      >       
+          <span v-if="isInCart({product})"> added to cart </span>
+      </li>
+    </ul>
+  <!-- ... -->
+</template>    
+<script>
```
Comment (filrak) [inline packages/core/docs/guide/cart-and-wishlist.md:163]: This is not how you would do this in a real app, I think the UI part can be removed (like it was)

File: packages/core/docs/guide/cart-and-wishlist.md [hunk 3]
```diff
@@ -383,22 +416,22 @@ To check if a product is already on the wishlist pass it to `isInWishlist` metho
 
 ```vue
 <template>
-  // ...
+   <!-- ... -->
```
Comment (filrak) [inline packages/core/docs/guide/cart-and-wishlist.md:419]: here UI part is obsolete, this is not how a real wishlist works so there is no benefit of having this to copy

**Review by filrak — CHANGES_REQUESTED** [review]
Also please correct formating (line breaks for tags with only 1 property)
```
<tag
   a="b"
>
</tag>
```
vs
```
<tag a="b">
</tag>
```

File: packages/core/docs/guide/cart-and-wishlist.md (hunk 2 above)
Comment (AdamPawlinski) [reply packages/core/docs/guide/cart-and-wishlist.md]: removed

File: packages/core/docs/guide/cart-and-wishlist.md (hunk 3 above)
Comment (AdamPawlinski) [reply packages/core/docs/guide/cart-and-wishlist.md]: removed

File: packages/core/docs/guide/cart-and-wishlist.md (hunk 1 above)
Comment (AdamPawlinski) [reply packages/core/docs/guide/cart-and-wishlist.md]: changed

**Review by filrak — APPROVED** [review]

**Pull request merged**